homeassistant/components/evohome.py

```diff
@@ -0,0 +1,144 @@
+"""Support for Honeywell evohome (EMEA/EU-based systems only).
+
+Support for a temperature control system (TCS, controller) with 0+ heating
+zones (e.g. TRVs, relays) and, optionally, a DHW controller.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/climate.evohome/
+"""
+
+# Glossary:
+# TCS - temperature control system (a.k.a. Controller, Parent), which can
+# have up to 13 Children:
+#   0-12 Heating zones (a.k.a. Zone), and
+#   0-1 DHW controller, (a.k.a. Boiler)
+
+import logging
+from requests.exceptions import HTTPError
```

Comment:
Add a blank line between standard library and 3rd party imports.

Reply:
Done